app/AccountancyModule/CampModule/templates/header.latte

```diff
@@ -1,4 +1,4 @@
-<h1 class="mt-5">{$event->getDisplayName()}</h1>
```

Comment:
Tohle odebíráš proč? Ten horní margin je to konzistentní se zbytkem UI.

Reply:
tedka to tam vratim pro konzistenci, ale je tam hrozně velká díra. A pokud ji tam chceme tak bych ji naopak udělat jednou jako mb-5 a ne u vsech nadpisu jako mt-5

Reply:
Tak ji tam pls vrať. Můžeme to používání pak nastavit jednou v samostatném PR. Možná pak nechat jako na jiných místech `.mt-4`, ale každopádně tam nemůže být margin nula - pak je to strašně přilepený:

## .mt-5
![image](https://user-images.githubusercontent.com/5658260/78456593-f546e680-76a4-11ea-8804-50a2e9c63e65.png)
 
## .mt-4
![image](https://user-images.githubusercontent.com/5658260/78456629-16a7d280-76a5-11ea-9da4-bbc4b9d2280b.png)

## žádný margin
![image](https://user-images.githubusercontent.com/5658260/78456646-232c2b00-76a5-11ea-9d00-4e76f43d95b2.png)

---
Mně osobně tam tak velkej margin dává smysl - jinde je margin 4, protože je tam ještě submenu

![image](https://user-images.githubusercontent.com/5658260/78456673-45be4400-76a5-11ea-9da7-32cea06a4eab.png)